openedx-unsupported/configuration
# Add CREDENTIALS_BASE_URL to MFE environment.

**Description**

Some of the micro-frontend apps expect `CREDENTIALS_BASE_URL` variable to be provided when building the frontend.
For example [app-frontend-profile](https://github.com/edx/frontend-app-profile/blob/f09ffdf75b1703ab56b1a4df3792cf7a9650032c/src/profile/ProfilePage.jsx#L47) uses this value to construct a link for the "View My Records" button on user's profile page.

**Test instructions**:
1. Log into the sandbox and go to the profile page.
2. Verify that the "View My Records" is present and points to https://fakecredentials.example.com/records (a fake URL the sandbox was set up with).

TODO: I'm building a sandbox with `MFE_CREDENTIALS_BASE_URL` set to a custom URL.

**Sandbox**:
- https://mfeplayground-master.opencraft.hosting/
Note: This sandbox was built with `MFE_CREDENTIALS_BASE_URL` set to `"https://fakecredentials.example.com"`.

**Related**:
- https://github.com/edx/frontend-app-profile/pull/421

**Reviewers**:



Make sure that the following steps are done before merging:
